imgts/entrypoint.sh

```diff
@@ -107,32 +107,81 @@ if is_release_branch_image $BUILDID; then
     SET_PERM=1
 fi
 
-if ((DRY_RUN)); then GCLOUD='echo'; fi
+if ((DRY_RUN)); then
+    GCLOUD='echo'
+    AWS='echo'
```

Comment:
It doesn't look like any of the `$GCLOUD`/`$AWS` incantations are logged, but in case some day they are, a future maintainer might appreciate something like
```
    GCLOUD="echo $GCLOUD"
    AWS="echo $AWS"
```
Otherwise, the only thing logged is the arguments. It might be possible for the maintainer to figure out "is this an aws or gcloud command" based on the context, and perhaps the args also, but why not be kind to them in anticipation?

Reply:
Sure thing, it's a good suggestion.